src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
+    <template v-if="$store.state.user">
+      <h2>You are logged in!</h2>
+      <div class="logout">
+        <button @click="logout">Logout</button>
+      </div>
+      <router-view></router-view>
+    </template>
+    <template v-else>
+      <div class="login">
+        <Auth />
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import Auth from './Auth';
+
+export default {
+  name: 'app',
+  components: {
+    Auth,
+  },
+  methods: {
+    logout() {
```

Comment:
This looks in the store and goes and gets some code related to logging out and then serves that code up up as a function that can be used in this page